src/Links/Link.tsx

```diff
@@ -0,0 +1,49 @@
+import React, {
+  FunctionComponent,
+  ReactElement,
+  ReactNode,
+  useContext,
+  MouseEventHandler,
+} from 'react';
+import styled, { ThemeContext } from 'styled-components';
+import { Link as ReactLink } from 'react-router-dom';
+import { BaseTheme } from '../Theme';
+
+export interface LinkProps {
+  /** Text or components to be displayed as a representation of the link */
+  children: ReactNode;
+  /** A string that represents the link path */
+  to: string;
+  /** Function to call on click event */
+  onClick: MouseEventHandler;
+}
+
+const StyledLink = styled(ReactLink)`
+  text-decoration: none;
+  color: ${({ theme }): string => (theme.color.background.info.medium)};
+  &:hover {
```

Comment:
Probably a good idea to also override the `:visited` pseudoclass to look the same as the normal links.

Reply:
Added :visited pseudoclass through 2c07673